src/lando/ui/jinja2/stack/partials/revision-status.html

```diff
@@ -1,4 +1,5 @@
-{% if revision['blocked_reason'] %}
+{% if revision['blocked_reasons'] or revision['blocked_reason'] %}
+{% set blocked_reason = revision['blocked_reasons'][0] if 'blocked_reasons' in revision else revision['blocked_reason']%}
```

Comment:
Ditto.

Reply:
This is the state in the upstream repo, I'm hoping to do some work to improve this after the Git migration is completed.